.travis.yml

```diff
@@ -36,3 +36,4 @@ notifications:
 
 services:
   - redis-server
+  - postgresql
```

Comment:
It turns out that Travis does not come with built-in postgresql anymore. Without this line, there will be no postgres running.